src/Helper/Scripts.php

```diff
@@ -1,68 +1,53 @@
 <?php
 /**
- *
  * This file is part of Aura for PHP.
  *
  * @license http://opensource.org/licenses/bsd-license.php BSD
- *
  */
 namespace Aura\Html\Helper;
 
 /**
- *
  * Helper for a stack of <script> tags.
  *
  * @package Aura.Html
- *
  */
 class Scripts extends AbstractSeries
 {
-    private $capture;
+    private $_capture;
```

Comment:
I'm not running any auto fixers or anything, but I've got vim set up to yell at me about stuff.
Why is this no good? Are the other Style fixes I just pushed a problem (lining up docblocks etc)?